MODULE.md

```diff
@@ -0,0 +1,262 @@
+
+    This command will fail if automatic inflation is enabled.
+
+  **Example:** `manifestd tx manifest stakeholder-payout 777umfx`
+
+## Proof of Authority Module
+
+The PoA module is responsible for handling admin actions like adding and removing other administrators, setting the staking parameters of the chain, controlling voting power, and allowing/blocking validators. Below is a structured breakdown of its components and functionalities:
+
+### Module Functionality
+
+The PoA admin has several capabilities for managing the chain and its validators:
+
+#### Validator Management:
+
+- Remove validators from the active set.
+- Remove validators pending addition to the active set.
+- Specify the voting power for each validator.
+- Approve the addition of new validators.
+
+#### Staking Parameters Update:
+
+- The PoA admin can update various staking parameters, including:
+  - `unbondingTime`: The time period for unbonding tokens.
+  - `maxVals`: The maximum number of validators.
+  - `maxEntries`: The maximum number of entries.
+  - `historicalEntries`: The number of historical entries to store.
+  - `bondDenom`: The denomination for bonding and staking.
+  - `minCommissionRate`: The minimum commission rate for validators.
+
+#### Administrative Rights:
+
+- Assign or revoke administrative privileges.
+- Determine if validators have the ability to self-revoke.
+
+#### Commands
+
+##### Update Parameters (update-params):
+
+- Syntax: `manifestd tx poa update-params [admin1,admin2,admin3,...] [allow-validator-self-exit-bool]`
+
+  - Parameters:
+    - `admin1,admin2,admin3,...`: A list of admin addresses which can be multisig addresses.
+    - `allow-validator-self-exit-bool`: A boolean value (true or false) to allow validators to self-exit.
+
+  **Example:** `manifestd tx poa update-params manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct,manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct false`
+
+##### Update Staking Parameters (update-staking-params):
+
+- Syntax: `manifestd tx poa update-staking-params [unbondingTime] [maxVals] [maxEntries] [historicalEntries] [bondDenom] [minCommissionRate]`
+
+  - Parameters:
+
+    - `unbondingTime`: The amount of time it takes to unbond tokens.
+    - `maxVals`: The maximum number of validators.
+    - `maxEntries`: The maximum number of entries.
+    - `historicalEntries`: The number of historical entries to store.
+    - `bondDenom`: The denomination for bonding and staking.
+    - `minCommissionRate`: The minimum commission rate for validators.
+
+  **Example:** `manifestd tx poa update-staking-params 1814400 100 7 1000 umfx 0.01`
+
+##### Set Voting Power (set-power):
+
+- Syntax: `manifestd tx poa set-power [validator] [power] [--unsafe]`
+
+  - Parameters:
+
+    - `validator`: The validator's operator address.
+    - `power`: The voting power to give the validator.
+    - `--unsafe`: A flag to allow the PoA admin to set the voting power of a validator that is not in the active set.
```

Comment:
What is the use-case for this flag?

Reply:
As is explained in the flag itself, the unsafe flag allows the PoA admin to set the voting power of a validator that is unactive, meaning they are jailed, or have had their voting power removed prior.

Reply:
Thanks. Can you document a flow where this is used?